task_spec/.fixtures.yml

```diff
@@ -1,3 +1,8 @@
 fixtures:
+  repositories:
+    facts:
+      repo: "git@github.com:donoghuc/puppetlabs-facts.git"
```

Comment:
The README should definitely note the dependency on puppetlabs-facts. I'm leaning against putting it in metadata.json yet because it's only required for tasks.